scipy/signal/_spectral_py.py

```diff
@@ -106,6 +108,9 @@ def lombscargle(
     any bias due to consistently missing observations at peaks and/or troughs. When the
     normalize parameter is "amplitude", for any frequency in freqs that is below
     ``(2*pi)/(x.max() - x.min())``, the predicted amplitude will tend towards infinity.
+    Unlike evenly-sampled data (required for FFT variants), when the data are unevenly
+    sampled, the upper frequency is not limited to the sampling rate divided by 2
+    (Nyquist-Shannon sampling theorem).
```

Comment:
Agreed. I had another go at tweaking the wording.

Reply:
I find it humorous that one of the issues to close (https://github.com/scipy/scipy/issues/8787) is only an issue specifically because their data is both *evenly sampled and has a freq that is a multiple of the "pseudo-Nyquist". If even one data point was multiplied by `(1.0 + eps)` (e.g., `x[0] *= (1.0 + eps)`) then the determinate is never zero, and the results are nearly identical (i.e., sum of the abs of the differences, except for that one frequency that has to be corrected, is 1.79e-8).

*Not evenly sampled in a way that FFT could be used. But in the sense that all of the data are sampled with the same dt, and the "gaps" are all integer multiples of the "normal" dt.